src/OutputPanel.vala

```diff
@@ -41,6 +41,10 @@ public class Sound.OutputPanel : Gtk.Grid {
         available_label.halign = Gtk.Align.START;
         devices_listbox = new Gtk.ListBox ();
         devices_listbox.activate_on_single_click = true;
+        devices_listbox.row_activated.connect (() => {
```

Comment:
the `row_activated` signal already returns the row so no need to use `devices_listbox.get_selected_row ()`

Reply:
Good point